api/HTMLIFrameElement.json

```diff
@@ -70,10 +70,10 @@
               "version_added": null
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
```

Comment:
This is not consistent with it being added in Chrome 43, so that data must be wrong. That's the case for a lot of changes in this PR which aren't in https://github.com/mdn/browser-compat-data/pull/7063, but I trust these changes based on them being based on testing in Opera 12.1, 15 and the latest version of Opera.